src/material/chips/testing/chip-list-harness.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness, HarnessPredicate} from '@angular/cdk/testing';
+import {MatChipHarness} from './chip-harness';
+import {MatChipInputHarness} from './chip-input-harness';
+import {
+  ChipListHarnessFilters,
+  ChipHarnessFilters,
+  ChipInputHarnessFilters,
+} from './chip-harness-filters';
+
+/** Harness for interacting with a standard chip list in tests. */
+export class MatChipListHarness extends ComponentHarness {
+  /** The selector for the host element of a `MatChipList` instance. */
+  static hostSelector = '.mat-chip-list';
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a `MatChipListHarness` that meets
+   * certain criteria.
+   * @param options Options for filtering which chip list instances are considered a match.
+   * @return a `HarnessPredicate` configured with the given options.
+   */
+  static with(options: ChipListHarnessFilters = {}): HarnessPredicate<MatChipListHarness> {
+    return new HarnessPredicate(MatChipListHarness, options);
+  }
+
+  /** Gets whether the chip list is disabled. */
+  async isDisabled(): Promise<boolean> {
+    return await (await this.host()).getAttribute('aria-disabled') === 'true';
+  }
+
+  /** Gets whether the chip list is required. */
+  async isRequired(): Promise<boolean> {
+    return await (await this.host()).getAttribute('aria-required') === 'true';
+  }
+
+  /** Gets whether the chip list is invalid. */
+  async isInvalid(): Promise<boolean> {
+    return await (await this.host()).getAttribute('aria-invalid') === 'true';
+  }
+
+  /** Gets whether the chip list is in multi selection mode. */
+  async isMultiple(): Promise<boolean> {
+    return await (await this.host()).getAttribute('aria-multiselectable') === 'true';
+  }
+
+  /** Gets whether the orientation of the chip list. */
+  async getOrientation(): Promise<'horizontal' | 'vertical'> {
+    const orientation = await (await this.host()).getAttribute('aria-orientation');
+    return orientation === 'vertical' ? 'vertical' : 'horizontal';
+  }
+
+  /**
+   * Gets the list of chips inside the chip list.
+   * @param filter Optionally filters which chips are included.
+   */
+  async getChips(filter: ChipHarnessFilters = {}): Promise<MatChipHarness[]> {
+    return this.locatorForAll(MatChipHarness.with(filter))();
+  }
+
+  /** Gets the selected chips from the chip list. */
+  async getSelectedChips(): Promise<MatChipHarness[]> {
```

Comment:
If we added `selected` to `ChipHarnessFilters` this method wouldn't be necessary, we could just use `getChips({selected: true})`